<ui version="4.0" >
 <class>NVideoPlayer</class>
 <widget class="QWidget" name="NVideoPlayer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Video Player</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <spacer name="spacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>381</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QFrame" name="videoBarFrame" >
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" name="_8" >
        <item>
         <spacer name="spacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="controlPadIconLabel" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="pixmap" >
           <pixmap resource="../../resources/videoplayer.qrc" >:/controlpad.png</pixmap>
          </property>
          <property name="scaledContents" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" name="_5" >
        <item>
         <spacer name="spacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>101</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="playStateFrame" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>120</width>
            <height>34</height>
           </size>
          </property>
          <property name="autoFillBackground" >
           <bool>true</bool>
          </property>
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="_6" >
           <property name="leftMargin" >
            <number>10</number>
           </property>
           <property name="topMargin" >
            <number>5</number>
           </property>
           <property name="rightMargin" >
            <number>5</number>
           </property>
           <property name="bottomMargin" >
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="stateIconLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize" >
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="stateRateLabel" >
             <property name="palette" >
              <palette>
               <active>
                <colorrole role="WindowText" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>41</red>
                   <green>41</green>
                   <blue>41</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="BrightText" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="HighlightedText" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="WindowText" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>41</red>
                   <green>41</green>
                   <blue>41</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="BrightText" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="HighlightedText" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="WindowText" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>115</red>
                   <green>115</green>
                   <blue>115</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="BrightText" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="HighlightedText" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="font" >
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="spacer_3" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>101</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <widget class="QFrame" name="playBarFrame" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground" >
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="_7" >
         <property name="leftMargin" >
          <number>20</number>
         </property>
         <property name="topMargin" >
          <number>10</number>
         </property>
         <property name="rightMargin" >
          <number>5</number>
         </property>
         <property name="bottomMargin" >
          <number>10</number>
         </property>
         <item>
          <widget class="NStorageIconLabel" name="storageIconLabel" >
           <property name="minimumSize" >
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="storageIcon" stdset="0" >
            <enum>NStorageIconLabel::IconOSD</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>16</width>
             <height>60</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="NTimeProgressBar" name="timeProgressBar" >
           <property name="palette" >
            <palette>
             <active>
              <colorrole role="WindowText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="BrightText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>192</red>
                 <green>192</green>
                 <blue>192</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Highlight" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>0</red>
                 <green>85</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="BrightText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>192</red>
                 <green>192</green>
                 <blue>192</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Highlight" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>0</red>
                 <green>85</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="BrightText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>192</red>
                 <green>192</green>
                 <blue>192</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Highlight" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>217</red>
                 <green>217</green>
                 <blue>217</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font" >
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="currentTextVisible" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="videoInfoLabel" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Maximum" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font" >
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NStorageIconLabel</class>
   <extends>QLabel</extends>
   <header>nstorageiconlabel.h</header>
  </customwidget>
  <customwidget>
   <class>NTimeProgressBar</class>
   <extends>QWidget</extends>
   <header>ntimeprogressbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/videoplayer.qrc" />
 </resources>
 <connections/>
</ui>
